README.md

```diff
@@ -92,9 +96,10 @@ $response = $matej->request()
     ->send();
 
 // Get list of item properties that are defined in matej
-$response = $matej->request()
+$properties = $matej->request()
     ->getItemProperties()
-    ->send();
+    ->send()
```

Comment:
I know its shorter, but I'd prefer splitting the examples into two variables, and don't mix the fluent interface like this (where the return type changes during the calls) , whats is IMO not the best practice and not really transparent.

```
$response = $matej->request()
     ->getItemProperties()
     ->send();

$properties = $response->getData();
```

Similarly for the other examples...
```
$response = $matej->request()
     ->recommendation(UserRecommendation::create('user-id', 5, 'test-scenario', 1.0, 3600))
     ->setInteraction(Interaction::purchase('user-id', 'item-id')) // optional
     ->setUserMerge(UserMerge::mergeInto('user-id', 'source-id')) // optional
     ->send();

$recommendedItems  = $response->getRecommendation()
    ->getData();
```

etc.?